bin/set-edoc-hidden

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+[ -f "$1" ] || exit 1
+
+FILE="$1"
+HIDDEN="%% @hidden"
+
+if [ "$(head -n 1 $FILE)" != "$HIDDEN" ]; then
+    echo "0a\n$HIDDEN\n.\nw" | ed "$FILE" >/dev/null
```

Comment:
Nice to see you adopted bin folder!
What the purpose of this script? it adds `@hidden` tag to `FILE`?